### PR TITLE
install dappgui and create shortcuts

### PR DESCRIPTION
**Pull request by ubozov**

### CODE REVIEW

File: dapp/dapp_windows.go [hunk 1]
```diff
@@ -95,3 +106,37 @@ func modifyDappConfig(configFile string, dbConf *data.DB) error {
...
+	scriptTxt.WriteString("sub CreateShortCut()\n")
+	scriptTxt.WriteString("dim objShell, objLink\n")
+	scriptTxt.WriteString("set objShell = CreateObject(\"WScript.Shell\")\n")
+	scriptTxt.WriteString("set objLink = objShell.CreateShortcut(\"")
+	scriptTxt.WriteString(destination + "\\" + linkName + ".lnk\")\n")
+	scriptTxt.WriteString("objLink.Arguments = \"" + arguments + "\"\n")
+	scriptTxt.WriteString("objLink.Description = \"Privatix Dapp\"\n")
+	scriptTxt.WriteString("objLink.TargetPath = \"" + target + "\"\n")
+	scriptTxt.WriteString("objLink.WindowStyle = 1\n")
+	scriptTxt.WriteString("objLink.WorkingDirectory = \"" + path + "\"\n")
```
[the remark this answers is not in the document]
Comment (ubozov) [reply dapp/dapp_windows.go]: updated

File: dapp/dapp_windows.go [hunk 2]
```diff
@@ -95,3 +106,40 @@ func modifyDappConfig(configFile string, dbConf *data.DB) error {
...
+	fileName := path.Base(d.DownloadGui)
+	path := d.InstallPath
+	target := path + fileName
+	extension := filepath.Ext(fileName)
+	linkName := fileName[0 : len(fileName)-len(extension)]
+	destination := util.DesktopPath()
+	arguments := ""
+
+	var scriptTxt bytes.Buffer
+	scriptTxt.WriteString(fmt.Sprintf(`
```
Comment (sofabeat) [inline dapp/dapp_windows.go]: 1. Shortcut seems to be created already when dappgui builds
2. If it is still required to use Win32 API, I would suggest to reference "dll" directly and not to rely on script execution. One reason maybe, that `vbs` scripts are blocked by group policy.

Comment (ubozov) [reply dapp/dapp_windows.go]: 1. I think, daapgui will be `exe` file. Will we build it during the installation?
2. I will research your suggestion.

File: dapp/dapp_windows.go [hunk 3]
```diff
@@ -106,3 +118,40 @@ func modifyDappConfig(configFile string, dbConf *data.DB, role string) error {
...
+	option explicit
+
+	sub CreateShortCut()
+		dim objShell, objLink
+		set objShell = CreateObject("WScript.Shell")
+		set objLink = objShell.CreateShortcut("%s\%s.lnk")
+		objLink.Arguments = "%s"
+		objLink.Description = "Privatix Dapp"
+		objLink.TargetPath = "%s"
+		objLink.WindowStyle = 1
```
Comment (drew2a) [inline dapp/dapp_windows.go]: From my perspective better to see here constant with an understandable name instead `1`

**Pull request merged**